src/Temporalio/Worker/WorkflowInstance.cs

```diff
@@ -146,13 +146,18 @@ public WorkflowInstance(WorkflowInstanceDetails details)
                     RunId: start.ParentWorkflowInfo.RunId,
                     WorkflowId: start.ParentWorkflowInfo.WorkflowId);
             }
+            var lastFailure = start.ContinuedFailure == null ?
+                null : failureConverter.ToException(start.ContinuedFailure, PayloadConverter);
+            var lastResult = start.LastCompletionResult?.Payloads_.Select(v => new RawValue(v)).ToArray();
```

Comment:
Are codecs automatically applied to this? I think you used some reflection magic somewhere to ensure we apply codecs to anything over the lang-core bridge.

Reply:
Yes, codecs are applied outside of this and there's a test to confirm no payloads slip by (even if they were just added to proto)